content/pt/docs/referencia-da-api/pix/apis-stone/reivindicacoes/reivindicar-uma-chave-pix/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Reivindicar"
```

Comment:
@leticiafiori it's the same endpoint to `Criar Reivindicação/Portabilidade`, so it's duplicated. It will be nice to concentrate the description [there](https://github.com/stone-co/stone-api-docs/pull/385/files#diff-71ee51e74980b22c5e6abee70fb93f4e08f521ec2bd315fa7f5340b01fefea42R2) :)